src/hooks/useForm.js

```diff
@@ -0,0 +1,13 @@
+import { React, useState } from 'react';
+
+export function useForm(inputs = {}) {
+  const [formState, setFormstate] = useState(inputs);
+
+  const handleFormChange = (event) => {
+    const { name, value } = event.target;
+    setFormstate((prevState) => {
+      return { ...prevState, [name]: value };
```

Comment:
I love the use of destructuring here! If you wanted to get super clever (albeit a bit code-golfy lol), you could take it a step further even; like this: 

```js
const handleFormChange = ({ target: { name, value }}) => {
    setFormState((prevState) => {
        return { ...prevState, [name]: value };
    });
};
```

_P.S. This isn't necessarily the best way to write it, it gets a bit tricky to read but it's still cool that it's possible! haha!_